.github/workflows/continuous.yml

```diff
@@ -120,6 +120,16 @@ jobs:
         with:
           name: tensorflow-model-zoo
           path: tensorflow/tensorflow-model-zoo/build/reports
+      - name: upload to codecov
+        if: always()
+        run: mkdir coverage && cp -t coverage/ $(find . -name "*jacocoTestReport.xml")
+      - uses: codecov/codecov-action@v1
+        with:
+          directory: ./coverage/
```

Comment:
With gradle task `jacocoRootReport`, it creates a joined jacocoTestReport. You should only need to use that one